fvtest/algotest/main.cpp

```diff
@@ -34,6 +34,8 @@ testMain(int argc, char **argv, char **envp)
 {
 	::testing::InitGoogleTest(&argc, argv);
 
+	OMREventListener::setDefaultTestListner();
```

Comment:
Typo: `s/listner/listener/`

Reply:
That would be a problem everywhere :)  I will update it and push again